docs/docson/build-schema.json

```diff
@@ -292,6 +319,10 @@
             "type" : "boolean",
             "description": "(experimental currently)default true(Set use OCaml stdlib or not)"
         },
+        "buildable-targets": {
+            "$ref": "#/definitions/buildable-targets",
```

Comment:
why is it either `js` or an array of targets instead of just an array of targets?